tests/Feature/UserTest.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+use App\Church;
+use App\User;
+
+class UserTest extends TestCase
+{
+  use RefreshDatabase;
+
+  protected $response;
+  protected $church;
+  protected $user;
+	
+  protected function setUp(): void
+  {
+    parent::setUp();
+    
+    $this->church = factory(Church::class)->create();
+    $this->response = $this->post(route('users.store'), [
+      'id' => 1,
```

Comment:
Similar to an earlier comment, it's better to not allow database id to be set externally.